test/plugins/reusePaths.06.svg.txt

```diff
@@ -1,5 +1,5 @@
 Don't remove and reuse the ID of the duplicate path if it's already being linked
-in an href by another node.
+in a href by another node.
```

Comment:
```suggestion
in an href by another node.
```

This one should be `an`! (href is pronounced aytch-ref, which starts with a vowel sound!)

Reply:
Ha, in my defence I pronounce it hay-ch-ref. Also, there's another instance of "a href" [here](https://github.com/svg/svgo/blob/main/plugins/prefixIds.js#L210) so I went for consistency.

Reply:
That's also wrong too then!

Can't speak for every country/culture, but in both US and UK, the uppercase character `H` does not start with a `h` sound.

Reference (this page includes an audio file for both UK and US pronunciations):  
https://dictionary.cambridge.org/us/pronunciation/english/h

Then, following this, if to use `a` vs `an` is based on the _sound_ of a vowel, but not the literal character used. So because an uppercase `H` starts with a vowel sound, it should be `an`. (The same reason why "It's an honor." is spelled with `an`, not `a`. `honor` does not start with a vowel character, but the way it's pronounced starts with a vowel sound.)

So href, assuming one pronounces `h` with the name of the uppercase letter, and then `ref` as in the first syllable of `reference`, is pronounced aytch-ref.

I'm not personally going to accuse you of pronouncing anything wrong, as I can see you're from Australia based on your GitHub profile, and it seems to vary there. But our docs are written in American English, so there is a single answer in our case!